api/v1/db/oracledb/parks-dao.js

```diff
@@ -277,10 +277,34 @@ const patchParkById = async (id, body) => {
   }
 };
 
+/**
+ * @summary Get park by owner Id
+ * @param {string} id Unique owner ID
+ * @returns {Promise<object>} Promise object represents a specific owner or return undefined if term
+ *                            is not found
+ */
+const getParkByOwnerId = async (id) => {
+  const sqlBinds = { ownerId: id };
+  const sqlQuery = `${getParkSql} AND OWNER_ID = :ownerId`;
+  const connection = await conn.getConnection();
+  try {
+    // check if owner exists
+    const ownerTest = await connection.execute('SELECT * FROM OWNERS WHERE ID = :ownerId', sqlBinds);
```

Comment:
Since you don't need any of the data for the owner you're querying for, could you use `SELECT COUNT(1)` here instead? This shouldn't have much of an effect in this case, but it's good practice for dealing with larger tables.